tests/integration/metrics/multi_column/test_sum_equal.py

```diff
@@ -0,0 +1,156 @@
+import pandas as pd
+import pytest
+from pyspark.sql.types import StringType
+
+from great_expectations.datasource.fluent.interfaces import Batch
+from great_expectations.metrics.multi_column.sum_equal import (
+    MultiColumnSumEqualUnexpectedCount,
+    MultiColumnSumEqualUnexpectedCountResult,
+)
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.metrics.conftest import (
+    PANDAS_DATA_SOURCES,
+    SPARK_DATA_SOURCES,
+    SQL_DATA_SOURCES,
+)
+from tests.integration.test_utils.data_source_config import SparkFilesystemCsvDatasourceTestConfig
+
+COL_A = "A"
+COL_B = "B"
+COL_A_WITH_NULLS = "A_WITH_NULLS"
+COL_B_WITH_NULLS = "B_WITH_NULLS"
+DATA_FRAME = pd.DataFrame(
+    {
+        COL_A: [1, 2, 3, 4],
+        COL_B: [4, 3, 2, 1],
+    },
+)
+SUCCESS_SUM_TOTAL = 5
+SUCCESS_COUNT = 0
+FAILURE_SUM_TOTAL = 0
+FAILURE_COUNT = 4
+DATA_FRAME_WITH_NULLS = pd.DataFrame(
```

Comment:
Can you add a case here where both are not None but don't sum to 5?

Reply:
i believe the `failure` test case below covers this, but let me know in case there's something else you're looking for